app/utils/asyncInjectors.js

```diff
@@ -33,6 +33,8 @@ export function injectAsyncReducer(store, isValid) {
       '(app/utils...) injectAsyncReducer: Expected `asyncReducer` to be a reducer function'
     );
 
+    if (Reflect.has(store.asyncReducers, name)) return;
```

Comment:
@mxstbr @smashercosmo sorry guys just getting on my computer... I have been using Reflect mainly as a first-class replacement for `in` and `delete` operators...

> In ES5, the way to detect whether an object obj defines or inherits a certain property name is to write (name in obj). Similarly, to delete a property, one uses delete obj[name]. While dedicated syntax is nice and short, it also means you must explicitly wrap these operations in functions when you want to pass the operation around as a first-class value.

> With Reflect, these operations are readily defined as first-class functions: Reflect.has(obj, name) is the functional equivalent of (name in obj) and Reflect.deleteProperty(obj, name) is a function that does the same as delete obj[name].

there are other good reasons to use Reflect api, check out the wiki: [Harmony `Reflect`](https://github.com/tvcutsem/harmony-reflect/wiki#harmony-reflect)